tests/Gedmo/Timestampable/DatePointTest.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+/*
+ * This file is part of the Doctrine Behavioral Extensions package.
+ * (c) Gediminas Morkevicius <gediminas.morkevicius@gmail.com> http://www.gediminasm.org
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Gedmo\Tests\Timestampable;
+
+use Doctrine\Common\EventManager;
+use Gedmo\Tests\Timestampable\Fixture\ArticleDatePoint;
+use Gedmo\Tests\Timestampable\Fixture\Author;
+use Gedmo\Tests\Timestampable\Fixture\CommentDatePoint;
+use Gedmo\Tests\Timestampable\Fixture\Type;
+use Gedmo\Tests\Tool\BaseTestCaseORM;
+use Gedmo\Timestampable\TimestampableListener;
+use Symfony\Component\Clock\DatePoint;
+use Symfony\Component\Clock\NativeClock;
+
+final class DatePointTest extends BaseTestCaseORM
```

Comment:
The tests in this class would need to be skipped if the date point type doesn't exist (i.e. someone runs the tests with Symfony 6.4 components loaded)